app/uk/gov/hmrc/gform/controllers/AuthenticatedRequestActions.scala

```diff
@@ -168,6 +180,12 @@ class AuthenticatedRequestActions(
       } yield result
     }
 
+  private def getAssumedIdentityCookie(cookie: Option[Cookie]): Option[String] =
+    cookie match {
```

Comment:
How was this resolved?

Reply:
This has been removed as it is no longer required after the refactor